src/packet_router/login_init.rs

```diff
@@ -14,25 +13,73 @@ pub fn init_login(
 ) {
     println!("Login protocol initiated :{:?}", p);
     match p.clone() {
-        Packet::LoginStart(packet) => {
-            println!("Joining ...");
-            let login_packet = packet::LoginSuccess {
-                uuid: "1c88a2d0-cdf1-4999-9dc1-3e2f696dde05".to_string(),
-                username: packet.username,
-            };
-            println!("{:?}", login_packet);
-            messenger
-                .send(MessengerOperations::Send(SendPacketMessage {
-                    conn_id,
-                    packet: Packet::LoginSuccess(login_packet),
-                }))
-                .unwrap();
-            *state = 3; // Switches to play state
-            println!("state={}", *state);
-            game_state::spawn_player(conn_id, messenger);
+        Packet::LoginStart(login_start) => {
+            *state = 3;
```

Comment:
We don't need to set the state to 3 after the login success packet- the point in this method where we change the state is irrelevant since this connection is single threaded and won't read from the socket again until this method is complete anyway